### ISSUE TITLE
Hyperlinks in Abstract shouldn't include trailing dot

### ISSUE THREAD

**Issue by sumothecat** [datagovuk/ckanext-dgu#284]
The plain-text hyperlink parser shouldn't include the trailing full-stop. For example, the abstract in this link https://data.gov.uk/dataset/habitat-point-records-from-1970-80-smba-mba-great-britain-littoral-survey.

In the previous sentence the full-stop wasn't considered by Github to part of the link. Obviously a URI can end in a dot, but in practice within plain text it's almost always a full-stop. We added a temporary redirect on the host website to make the link in the abstract work, but we'd like to remove this if possible.

**Comment by davidread** [comment]
I agree, but this is a 'nice-to-have'. We're trying a new markdown parser shortly, so maybe that'll sort it. However it is not a priority for us to hack any markdown parsers.

**Comment by rossjones** [comment]
Still happens with new markdown parser, so will leave it open for now.
